include/highfive/bits/H5Node_traits.hpp

```diff
@@ -110,7 +110,38 @@ class NodeTraits {
     ///
     /// \brief return the name of the object with the given index
     /// \return the name of the object
-    std::string getObjectName(size_t index) const;
+    std::string getName(size_t index) const;
+
+    ///
+    /// \brief return the path to the current object
+    /// \return the path to the object
+    std::string getPath() const;
+
+    ///
+    /// \brief moves an object and its content within an HDF5 file. Moved object is still valid but content - NOT!
```

Comment:
```suggestion
    /// \brief moves an object and its content within an HDF5 file.
```

I don't understand this remark. I have the feeling that it is not entirely true either.